web/src/components/RemoteContentSandBox.ts

```diff
@@ -144,6 +144,38 @@ const RemoteContentSandBox = styled.div<{ $centered: boolean; $smallText: boolea
     font-weight: bold;
   }
 
+  .contact-card {
+    display: inline-block;
+    box-sizing: border-box;
+    padding: 1rem;
+    border-radius: 0.25rem;
+    background-repeat: no-repeat;
+    background-color: rgb(127 127 127 / 15%) !important;
+    background-image: linear-gradient(to right, rgb(255 255 255 / 90%) 0 100%),
+      url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyNCIgaGVpZ2h0PSIyNCIgdmlld0JveD0iMCAwIDI0IDI0Ij48cGF0aCBkPSJNMTMuMDE4IDEyLjQ4aC0yLjAzNkE5LjA5IDkuMDkgMCAwIDAgMS45MiAyMS42YS40OC40OCAwIDAgMCAuNDguNDhoMTkuMmEuNTMuNTMgMCAwIDAgLjQ4LS41MzggOS4wOCA5LjA4IDAgMCAwLTkuMDYyLTkuMDYyTTE2LjggNi43MmE0LjggNC44IDAgMCAxLTQuOCA0LjggNC44IDQuOCAwIDAgMS00LjgtNC44IDQuOCA0LjggMCAwIDEgNC44LTQuOCA0LjggNC44IDAgMCAxIDQuOCA0LjgiLz48L3N2Zz4=') !important;
+    background-blend-mode: difference;
+    background-position: calc(100% + 2em) calc(100% + 1em);
+    background-size: 7em;
+    box-shadow: 0 0.1em 0.1em rgb(0 0 0 / 40%);
+    min-width: 72%;
+
+    p {
+      margin: 4px 0 0;
```

Comment:
:upside_down_face: Could you try to simplify the css a bit still? This is the same as the following, right? Same for other css please :) 
```suggestion
      margin-top: 4px;
```